app/views/trainees/personal_details/edit.html.erb

```diff
@@ -1,15 +1,15 @@
 <%= content_for(:breadcrumbs) do %>
   <%= render_component GovukComponent::BackLink.new(
     text: 'Back',
-    href: trainee_path(@trainee)
+    href: trainee_path(@personal_details_form.trainee)
   ) %>
 <% end %>
 
 <h2 class="govuk-heading-l">Trainee personal details</h2>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <%= form_with model: @trainee, local: true do |f| %>
+    <%= form_for(@personal_details_form, url: trainee_personal_details_path(@personal_details_form.trainee), method: :put, local: true) do |f| %>
       <%= f.govuk_text_field :first_names, label: { text: 'First names', size: 's' }, hint: { text: 'Or given names' }, width: 'three-quarters' %>
```

Comment:
Not sure within remit of ticket but may be worth putting in an error summary at [top of page](https://govuk-form-builder.netlify.app/introduction/error-handling/#input-erb-a-text-field-with-an-error) for users

Reply:
done